boards/arm/frdm_kw41z/Makefile.board

```diff
@@ -2,4 +2,6 @@ DEBUG_SCRIPT = jlink.sh
 
```

Comment:
The docs already do talk about the different OpenSDA firmwares. See doc/tools/opensda.rst